netbox_topology_views/views.py

```diff
@@ -225,6 +225,7 @@ def create_edge(
         edge["dashes"] = [1, 10, 1, 10]
         edge["arrows"] = {"to": {"enabled": True, "scaleFactor": 0.5}, "from": {"enabled": True, "scaleFactor": 0.5}}
         edge["color"] = '#f1c232'
+        edge["href"] = interface.get_absolute_url() + "/trace"
```

Comment:
```suggestion
        edge["href"] = interface.get_absolute_url() + "trace"
```
The trailing slash renders to e.g. "...interfaces/375//trace" (note the double slash) which leads to a 404 in my test environment.

Reply:
Your suggestion works fine in my environment too.